src/core/manager.h

```diff
@@ -475,6 +475,9 @@ struct Manager {
         RateLimit dump_ratelimit;
 
         sd_event_source *memory_pressure_event_source;
+
+        /* Instance counts of template units with MaxInstance= set */
```

Comment:
maybe say explicitly that the key here is the template name and the value the count